### ISSUE TITLE
"Oops. We could not send the error report."

### ISSUE THREAD

**Issue by janpio** [prisma/prisma2#1869]
1. Clone https://github.com/harshit-test-org/prisma2-client-502
1. Update `package.json` to `preview024`
2. `yarn`
3. Adapt `prisma/.env` to point to local MySQL 5.7 installation
3. `yarn prisma2 migrate save --experimental`
4. `yarn prisma2 migrate up --experimental`
5. See:
	```
	You can get the detailed db changes with prisma2 migrate up --experimental --verbose
	Or read about them in the ./migrations/MIGRATION_ID/README.md
	 ERROR  Oops, an unexpected error occured!
	Failure during a migration command: Connector error. (error: Error querying the database: Error querying the database: Error querying the database: Server error: `ERROR 42S02 (1146): Table '_issue_client_502._migration' doesn't exist')
	Please help us improve Prisma 2 by submitting an error report.
	Error reports never contain personal or other sensitive information.
	Learn more: https://pris.ly/d/telemetry
	
	┌─ Submit error report ─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐  
	│                                                                                                                                                                                                                                                                                               │  
	│ > Yes            Send error report once                                                                                                                                                                                                                                                       │  
	│   No             Don't send error report                                                                                                                                                                                                                                                      │  
	│                                                                                                                                                                                                                                                                                               │  
	└───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘ 
	```
	after trying to send report:
	```
	...
	You can get the detailed db changes with prisma2 migrate up --experimental --verbose
	Or read about them in the ./migrations/MIGRATION_ID/README.md
	Failure during a migration command: Connector error. (error: Error querying the database: Error querying the database: Error querying the database: Server error: `ERROR 42S02 (1146): Table '_issue_client_502._migration' doesn't exist')
	Oops. We could not send the error report.
	
	Thanks a lot for your help! �🙏
	error Command failed with exit code 1.
	info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command
	```

(preview022 + preview024)

**Comment by pantharshit00** [comment]
Unable to reproduce with the provided reproduction. Migration just succeeds for me:
![image](https://user-images.githubusercontent.com/22195362/77261394-3528c980-6cb4-11ea-94b9-aca3b751e80a.png)

**Comment by janpio** [comment]
Your reproduction indicates you are not using Windows.

**Comment by pantharshit00** [comment]
I got a better reproduction for this. Try migrating the following model(issue: https://github.com/prisma/prisma2/issues/1860):
```prisma
model User {
  id   Int    @id @default(autoincrement())
  name String @unique

 @@index([name])
}
```

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/22195362/77308608-422fd200-6d21-11ea-9b5d-405e11759443.png">

**Comment by pantharshit00** [comment]
Not a windows specific issue but it indicates error reporting backend is broken or cli is sending faulty error reports.

**Comment by pantharshit00** [comment]
I am assuming this issue is tracking the fault of error reporting backend. 

The actual migrate issue is on my list now to test on windows.

**Comment by janpio** [comment]
The migrate issue is irrelevant here, you can ignore that and do not need to spend time on it - this is really just about the failure to send error reports.

**Comment by Jolg42** [comment]
```
2020-03-24T15:28:48.148Z sendPanic TypeError: Cannot read property 'signature' of undefined
    at Object.<anonymous> (/Users/j42/Dev/prisma2-client-502/node_modules/prisma2/build/index.js:2:1662033)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/j42/Dev/prisma2-client-502/node_modules/prisma2/build/index.js:2:1660593)
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
[?25h[2K[1G[31merror[39m Command failed with exit code 1.
```

How to find this logs?!
Well the interactive console for sending the reports hides it...
So `env DEBUG="*" yarn prisma2 migrate up --experimental 2> hidden-error.txt` 😉 

Fixed by 5b4b350940bf4015318572a63dc44124f51e5472 in alpha

Can't reproduce in 953! 🎊
<img width="708" alt="Screen Shot 2020-03-24 at 16 33 33" src="https://user-images.githubusercontent.com/1328733/77445433-622cc780-6ded-11ea-98f0-809f0648d017.png">

**Comment by janpio** [comment]
Can confirm that sending error reports now works.
